README.md

```diff
@@ -41,17 +41,21 @@ The configuration for openwisp images using environment variables is available [
 The following are steps of a sample deployment on a kubernetes cluster. All the files are present in `kubernetes/` directory of this repository.
 The following assumes the reader knows basics of kubernetes.
 
-1. (optional) Setup a Kubernetes Cluster: A guide for setting up the cluster on bare-metal machines is available [here](https://blog.alexellis.io/kubernetes-in-10-minutes/) and the guide to get started with kubernetes-dashboard (Web UI) is available [here](https://github.com/kubernetes/dashboard).
+If you are a looking for a guide specifically for Google Cloud Platform, please click [here](docs/GCP.md).
 
-2. Changes for your cluster:
+1. Setup a Kubernetes Cluster
+
+   1.1. A guide for setting up the cluster on bare-metal machines is available [here](https://blog.alexellis.io/kubernetes-in-10-minutes/) and the guide to get started with kubernetes-dashboard (Web UI) is available [here](https://github.com/kubernetes/dashboard).
 
-   2.1. `externalIPs` in `Service.yml` should to be your cluster's `externalIPs`
+   1.2. On providers you may initiate a cluster as per your desired configurations. (Read the [GCP cluster requirement](docs/GCP.md) to match those requirements in your cluster)
 
-   2.2 `ingress` in `Ingress.yml` should to be your cluster's loadbalancer IPs.
+2. Changes for your cluster:
```

Comment:
Need to add info regarding updating all domain names.  Domain names currently need to be configured in Ingress.yml, ExtServices.yml, Configmap.yml, Certificates.yml.  Perhaps we should make all other files pull from the Configmap.yml so that domain names only need to be set in one place.